### PR TITLE
fix(feel-popup): do not close popup on escape during auto suggest

### LINKED ISSUE #279: Can't close code suggestions in the FEEL popup editor
### Describe the Bug

Users can't close code suggestions in the FEEL popup editor with esc, since hitting the esc key closes the editor.

### Steps to Reproduce

1. Open FEEL popup editor
2. Type something to trigger a code suggestion, such as ctrl+space
3. Try to close the code suggestion by hitting the esc key
4. --> The full editor closes

### Expected Behavior

When a code suggestion is shown, hitting esc should close the suggestion, not the editor. The editor should close on esc only when no suggestion is shown.

### Environment

- Properties Panel version 3.4.0

### PR DESCRIPTION
**Pull request by pinussilvestrus**
Closes #279

### CODE REVIEW

**Review by vsgoulart — APPROVED** [review]

File: test/spec/components/FeelPopup.spec.js [hunk 1]
```diff
@@ -116,15 +116,15 @@ describe('<FeelPopup>', function() {
     });
 
     // assume
-    expect(domQuery('.bio-properties-panel-feel-editor-container', document.body)).to.exist;
+    expect(getFeelEditor(document.body)).to.exist;
```
Comment (nikku) [inline test/spec/components/FeelPopup.spec.js:119]: :star:

**Comment by pinussilvestrus** [comment]
@bpmn-io/modeling-dev I'd like to have another 👍 from one core developer before merging it 🙂

**Comment by barmac** [comment]
I will look into this today.

File: src/components/entries/FEEL/FeelPopup.js [hunk 2]
```diff
@@ -92,17 +92,34 @@ function FeelPopupComponent(props) {
...
+      // we need to do check this because the editor is not
+      // stop propagating the keydown event
+      // cf. https://discuss.codemirror.net/t/how-can-i-replace-the-default-autocompletion-keymap-v6/3322/5
+      if (!isAutoCompletionOpen.current) {
+        onClose();
+        isAutoCompletionOpen.current = false;
+      }
     }
-  }, [ editorRef, id ]);
+  };
```
Comment (barmac) [inline src/components/entries/FEEL/FeelPopup.js:122]: I thought we could simplify this by checking `event.defaultPrevented` value as it was suggested in https://discuss.codemirror.net/t/how-can-i-replace-the-default-autocompletion-keymap-v6/3322/7
However, a debug session revealed that the library `focus-trap` calls `preventDefault` in between. So we'd need to switch from `preact` event handler to vanilla javascript, which defies the simplification.

**Review by barmac — APPROVED** [review]
Let's merge this.

**Comment by pinussilvestrus** [comment]
I reported sick leave these days so feel free to take over to not block the integration 👍

**Pull request merged**